github/table_github_repository_content.go

```diff
@@ -0,0 +1,99 @@
+package github
+
+import (
+	"context"
+	"github.com/google/go-github/v45/github"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableGitHubRepositoryContent() *plugin.Table {
+	return &plugin.Table{
+		Name:        "github_repository_content",
+		Description: "List the content in a repository (list directory, or get file content",
+		List: &plugin.ListConfig{
+			Hydrate: tableGitHubRepositoryContentList,
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "repository_full_name", Require: plugin.Required},
+				{Name: "repository_content_path", Require: plugin.Optional, CacheMatch: "exact"},
+			},
+		},
+		Columns: []*plugin.Column{
+			{Name: "repository_full_name", Description: "The full name of the repository (login/repo-name).", Type: proto.ColumnType_STRING, Transform: transform.FromQual("repository_full_name")},
+			{Name: "type", Description: "The file type (directory or file).", Type: proto.ColumnType_STRING},
+			{Name: "name", Description: "The file name.", Type: proto.ColumnType_STRING},
+			{Name: "repository_content_path", Description: "The requested path in repository search.", Type: proto.ColumnType_STRING, Transform: transform.FromQual("repository_content_path")},
+			{Name: "path", Description: "The path of the file.", Type: proto.ColumnType_STRING},
+			{Name: "size", Description: "The size of the file.", Type: proto.ColumnType_INT},
```

Comment:
```suggestion
			{Name: "size", Description: "The size of the file(in MB).", Type: proto.ColumnType_INT},
```

Reply:
```suggestion
			{Name: "size", Description: "The size of the file (in MB).", Type: proto.ColumnType_INT},
```

I think we need a space beforehand